sqlalchemy_ibmi/base.py

```diff
@@ -19,165 +19,206 @@
-    'cursor', 'index', 'plan', 'union', 'cycle', 'indicator', 'position', 'unique', 'data',
-    'inherit', 'precision', 'until', 'database', 'inner', 'prepare', 'update',
-    'datapartitionname', 'inout', 'prevval', 'usage', 'datapartitionnum',
-    'insensitive', 'primary', 'user', 'date', 'insert', 'priqty', 'using', 'day',
-    'integrity', 'privileges', 'validproc', 'days', 'intersect', 'procedure', 'value',
-    'db2general', 'into', 'program', 'values', 'db2genrl', 'is', 'psid', 'variable',
-    'db2sql', 'isobid', 'query', 'variant', 'dbinfo', 'isolation', 'queryno', 'vcat',
-    'dbpartitionname', 'iterate', 'range', 'version', 'dbpartitionnum', 'jar', 'rank',
-    'view', 'deallocate', 'java', 'read', 'volatile', 'declare', 'join', 'reads', 'volumes',
-    'default', 'key', 'recovery', 'when', 'defaults', 'label', 'references', 'whenever',
-    'definition', 'language', 'referencing', 'where', 'delete', 'lateral', 'refresh',
-    'while', 'dense_rank', 'lc_ctype', 'release', 'with', 'denserank', 'leave', 'rename',
-    'without', 'describe', 'left', 'repeat', 'wlm', 'descriptor', 'like', 'reset', 'write',
-    'deterministic', 'linktype', 'resignal', 'xmlelement', 'diagnostics', 'local',
-    'restart', 'year', 'disable', 'localdate', 'restrict', 'years', '', 'abs', 'grouping',
-    'regr_intercept', 'are', 'int', 'regr_r2', 'array', 'integer', 'regr_slope',
-    'asymmetric', 'intersection', 'regr_sxx', 'atomic', 'interval', 'regr_sxy', 'avg',
-    'large', 'regr_syy', 'bigint', 'leading', 'rollup', 'blob', 'ln', 'scope', 'boolean',
-    'lower', 'similar', 'both', 'match', 'smallint', 'ceil', 'max', 'specifictype',
-    'ceiling', 'member', 'sqlexception', 'char_length', 'merge', 'sqlstate',
-    'character_length', 'method', 'sqlwarning', 'clob', 'min', 'sqrt', 'coalesce', 'mod',
-    'stddev_pop', 'collate', 'module', 'stddev_samp', 'collect', 'multiset',
-    'submultiset', 'convert', 'national', 'sum', 'corr', 'natural', 'symmetric',
-    'corresponding', 'nchar', 'tablesample', 'covar_pop', 'nclob', 'timezone_hour',
-    'covar_samp', 'normalize', 'timezone_minute', 'cube', 'nullif', 'trailing',
-    'cume_dist', 'numeric', 'translate', 'current_default_transform_group',
-    'octet_length', 'translation', 'current_role', 'only', 'treat',
-    'current_transform_group_for_type', 'overlaps', 'true', 'dec', 'overlay',
-    'uescape', 'decimal', 'percent_rank', 'unknown', 'deref', 'percentile_cont',
-    'unnest', 'element', 'percentile_disc', 'upper', 'exec', 'power', 'var_pop', 'exp',
-    'real', 'var_samp', 'false', 'recursive', 'varchar', 'filter', 'ref', 'varying',
-    'float', 'regr_avgx', 'width_bucket', 'floor', 'regr_avgy', 'window', 'fusion',
-    'regr_count', 'within', 'asc'])
-
-
-class _IBM_Boolean(sa_types.Boolean):
-
-    def result_processor(self, dialect, coltype):
+RESERVED_WORDS = {'activate', 'disallow', 'locale', 'result', 'add',
```

Comment:
Does it really format things this way instead of putting things on a new line and leaving them less indented?
Personally, I'd prefer these all go on separate lines to make diffs easier when things are changed/added. This will significantly increase the number of lines, though. We could mitigate that somewhat by putting it in its own module and just importing it here.